wg-api-expression/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - wg-api-expression-leads
+approvers:
+  - wg-api-expression-leads
+labels:
+  - wg/api-expression
```

Comment:
you'll want to open a PR against https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml to ensure this label exists

Reply:
Yeah, I was looking where this goes. Thanks for the link.